src/pudl/metadata/resources/eia861.py

```diff
@@ -694,6 +694,47 @@
         "sources": ["eia861"],
         "etl_group": "eia861",
     },
+    "core_eia861__yearly_short_form": {
+        "description": (
+            """This is data extracted from Form EIA-861S, a shorter version of Form EIA-861.
+The data started being reported in 2012. However, the 2019 data is not available.
+They are expected to submit the completed Form EIA-861S to EIA by April 30th, following the end of the prior calendar year.
+Utilities report on Form EIA-861S if they:
+- Report less than 200,000 megawatthours on the last previous Form EIA-861.
+- Provide only bundled service (generation and distribution).
+- Are not needed to ensure acceptable quality of statistical estimates.
+- Are not part of the aggregate TVA or WPPI.
+- Do not report on Form EIA-861M."""
```

Comment:
This bulletized list will not format correctly in RST. It'll all get run together in a single paragraph. Take a look at the output in the docs build (look at this table in the PUDL data dictionary) to debug.  I think you'll need to insert an empty line between the paragraph above and the bullets.